cmd/validator/credentials/set/process.go

```diff
@@ -238,10 +252,14 @@ func (c *command) generateOperationsFromMnemonic(ctx context.Context) error {
 	maxDistance := 1024
 	// Start scanning the validator keys.
 	lastFoundIndex := 0
+	found_validator_count := 0
```

Comment:
Please use camel case for variables; this will fail linting.

Reply:
Done.